.github/workflows/release.yml

```diff
@@ -107,9 +107,46 @@ jobs:
           name: bins-${{ matrix.build }}
           path: dist
 
+  source:
+    name: Create Source tarball
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          sudo apt-get update -qq -y
+          sudo apt-get install -qq -y python3-pip python3
+          sudo pip3 install git-archive-all
```

Comment:
I don't like the extra dependencies here. Since the submodules are already checked out it should be enough to package `helix-syntax/languages/*`. Let's hold off on this until we decide on #1560 though

Reply:
Yeah, for this case it should hopefully be simple enough to use tar with --exclude-vcs